### PR TITLE
Bump Vert.x version 4.3.4

### PR DESCRIPTION
**Pull request by ppatierno**
Bump Vert.x version to align with the one used by the operator.

### CODE REVIEW

**Review by scholzj — COMMENTED** [review]
I'm fine with this ... but would 4.3.5 make more sense? Or is there some reason to stick with 4.3.4?

**Comment by ppatierno** [comment]
First, I didn't notice 4.3.5 was out :-D ... second I thought it was ok aligning with operator but at this point we could align both to 4.3.5. Wdyt?

**Comment by scholzj** [comment]
I would go for the latest.

**Comment by ppatierno** [comment]
Fine with me, bumping to the latest.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 * Moved from using the Jaeger exporter to OTLP exporter by default
 * Moved to Java 11 at language level, dropped Java 8 and use Java 17 as the runtime for all containers
+* Dependency updates (Vert.x 4.3.4)
```
Comment (scholzj) [inline CHANGELOG.md:7]: I guess you can update this as well ;-) ... sorry :-o

```suggestion
* Dependency updates (Vert.x 4.3.5)
```

**Review by scholzj — APPROVED** [review]
One nit, LGTM otherwise

**Pull request merged**